flake8_isort.py

```diff
@@ -5,11 +5,19 @@
 from pathlib import Path
 
 import isort
+from pkg_resources import DistributionNotFound, get_distribution
+
+
+def _version():
+    try:
+        return get_distribution("flake8_isort").version
+    except DistributionNotFound:
+        return "dev"  # for local development if package is not installed yet
```

Comment:
Please, use single quotes, I very much prefer them, and try to keep consistency across the files 😄 

CI will probably fail due to that 😅

Reply:
Ah yeah right, changed it now :smile: